docs/setuptools.txt

```diff
@@ -2293,6 +2293,44 @@ Metadata and options are set in the config sections of the same name.
 
 * Unknown keys are ignored.
 
+setup.cfg-only projects
+=======================
+
+.. note:: New in 40.9.0 (03 Apr 2019).
```

Comment:
I think it's better to use `..versionadded` for this, and to add a changelog entry.

Reply:
I copied this style from other places in this document. Isn't consistency across the file important? Sure I can change this if you think that it's in the scope of this PR.